### DISCUSSION TITLE
Pregunta de cuantas practicas tenemos

### DISCUSSION THREAD

**Discussion by JorgeERG** [Launch-X-Latam/MisionFrontEnd#302] [General]
Hola, hago esta pregunta para confirmar cuantas practicas tenemos por el momento. es que no quiero que se me vaya a olvidar alguna

**Comment by iam-leo** [comment]
Hola, cada módulo tiene su práctica. Hasta el momento llevamos 3 Módulos vistos estos son:
* [Programación FrontEnd](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO)
* [HTML](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML)
* [CSS](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/03%20-%20CSS)

Actualmente estamos viendo el módulo de [JavaScript](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/04%20-%20JS) aún no hemos visto la totalidad de los contenidos de este módulo, pero eso no significa que no puedas adelantar la práctica de este o de los siguientes módulos, llévalo a tu ritmo.

**Reply by stvnmc** [reply]
la practica de css tiene fecha para marzo esa informacion esta bien ?

**Reply by iam-leo** [reply]
No, esas fechas son de una edición anterior de Launch X, ya lo irán a modificar o nos avisará el MC supongo!

**Reply by PatricioNovellinoLaunchx** [reply]
> Hola, cada módulo tiene su práctica. Hasta el momento llevamos 3 Módulos vistos estos son:
> * [Programación FrontEnd](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO)
> * [HTML](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML)
> * [CSS](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/03%20-%20CSS)
> 
> Actualmente estamos viendo el módulo de [JavaScript](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/04%20-%20JS) aún no hemos visto la totalidad de los contenidos de este módulo, pero eso no significa que no puedas adelantar la práctica de este o de los siguientes módulos, llévalo a tu ritmo.

La práctica de programación frontend (la primera) no tiene fecha de entrega, esa también hay que entregarla el 4?

**Reply by iam-leo** [reply]
En el live de ayer `02/11` el MC dijo que no hay fecha de entrega, solo hay que ir subiendo las prácticas en Github. Puedes volver a ver el live y en el minuto `09:00` aprox menciona esto de las fechas!

**Reply by PatricioNovellinoLaunchx** [reply]
> En el live de ayer `02/11` el MC dijo que no hay fecha de entrega, solo hay que ir subiendo las prácticas en Github. Puedes volver a ver el live y en el minuto `09:00` aprox menciona esto de las fechas!

Mil gracias! yo estoy un poco atrasado 1 o 2 clases por tema de estudio o trabajo

**Reply by iam-leo** [reply]
> > En el live de ayer `02/11` el MC dijo que no hay fecha de entrega, solo hay que ir subiendo las prácticas en Github. Puedes volver a ver el live y en el minuto `09:00` aprox menciona esto de las fechas!
> 
> Mil gracias! yo estoy un poco atrasado 1 o 2 clases por tema de estudio o trabajo

No te preocupes, yo también ando atrasado espero poder ponerme al día este finde jajaja
